content/en/technical-advisory-group/architectural-review-committee/ARC-001.en.md

```diff
@@ -7,4 +7,163 @@ draft: false
+- Centralized **GitOps pipelines (Argo CD)** for deploying and enforcing platform-level policies across all Aurora clusters
+
+### 4.2 Supporting Services and Management Clusters
+
+In addition to the platform clusters above, the Aurora model includes several **supporting service layers**:
+
+- **Development Services Environments** (NonProd and Prod): Dedicated to hosting shared services like **Artifactory**, **X-Ray**, **Backstage**, and others critical to Aurora's developer experience
+- **[Planned] Shared Management Cluster**: A centralized Argo CD environment to orchestrate deployment pipelines and policy enforcement across all Aurora clusters.
+  - *Problem to solve:* Ensuring secure, scalable **connectivity from this management plane to all clusters** across both the Enterprise Tenant and departmental landing zones
+- **[Planned] Shared SDLC Environments**: Hosted in the Enterprise Tenant, enabling **small departments and agencies** to access secure CI/CD pipelines and runtime environments without managing their own clusters
+- **[Planned] Per-Department SDLC Clusters**: Deployed either in the **Enterprise Tenant** (preferred) or **departmental tenants**, with connectivity to departmental landing zones. These clusters allow departments with more advanced needs to onboard in a governed but autonomous manner
+
+### 4.3 Team Structure
+
+To support and operate this model, Aurora introduces clearly defined roles:
+
+- **Architecture & Governance Team**: Defines technical standards, compliance boundaries, and platform blueprints
+- **Platform Operations Team**: Manages the lifecycle of core horizontal services (e.g., GitLab, Vault, Artifactory, CI/CD)
+- **Advisory TAGs** (*Architecture*, *Networking*, *Security*): Serve as expert panels to guide technical decisions, validate designs, and drive community alignment
+
+### Key Decision Point for ARC
+
+✔️ Does this operational model align with SSC’s cloud-native strategy?
+✔️ Does the structure support long-term scalability across departments and tenants?
+
+---
+
+## 5. Risks & Mitigations
+
+| **Risk**                           | **Mitigation Strategy**                                                                           |
+| ---------------------------------- | ------------------------------------------------------------------------------------------------- |
+| Departments resist standardization | Aurora allows **flexible adoption** while enforcing key governance policies.                      |
+| OSS vs Proprietary Tensions        | Supports **both managed OpenShift and Kubernetes**, providing departments with options.           |
+| Security & Compliance Overhead     | **Automated policy enforcement** via Gatekeeper, OPA, and workload isolation.                     |
+| Multi-cloud Complexity             | **Standardized onboarding, automated provisioning**, and governance frameworks reduce complexity. |
```

Comment:
I think another mitigation to your multi-cloud strategy is having experts of the different cloud providers in GC Cloud One assigned to working with Aurora for development within that cloud environment.